src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -190,8 +192,14 @@ describe('<PillButton> component', () => {
         });
 
         it('should have arrow up label', () => {
-            expect(screen.getByRole(imgRole)).toBeInTheDocument();
-            expect(getButtonByName(upArrowLabel)).toBeInTheDocument();
+            expect(screen.getByRole(imgRole, { name: upArrowLabel })).toBeVisible();
```

Comment:
this and the next line is the very same check, only once you call it through the function and once inline, am I correct (in which case this line can be removed)?

Reply:
We have img and button with 'arrow up' label. I think we need to check both

Reply:
I missed that. Thanks.